.golangci.yml

```diff
@@ -0,0 +1,52 @@
+# options for analysis running
+run:
+  # default concurrency is a available CPU number
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  timeout: 5m
+linters:
+  # please, do not use `enable-all`: it's deprecated and will be removed soon.
+  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
+  disable-all: true
+  enable:
+    - typecheck
+    - govet
+    - gosec
+    - gofmt
+    - golint
```

Comment:
I find it very suspicious that you're not getting errors from golint given the current exclusions below. Usually `thrift-gen` produces a lot of errors for golint, which is why we excluded it in the makefile